packages/venia-ui/lib/queries/getCustomer.graphql

```diff
@@ -0,0 +1,8 @@
+query getCustomer {
```

Comment:
I'd add a comment here indicating that this query depends on the `bearer` token being present otherwise it won't return anything (or it may just fail?)

Reply:
Fixed by d9eff35